README.md

```diff
@@ -25,7 +25,7 @@ Usage
 
 SuperCollider consists of three separate components:
 
-1. scsynth - audio engine (the "server")
+1. scsynth (and supernova) - audio engine (the "server")
```

Comment:
Maybe "scsynth or supernova" to convey the idea that you use one or the other, but not really both at once